components/elm/bld/ELMBuildNamelist.pm

```diff
@@ -2321,6 +2321,7 @@ sub setup_logic_demand {
   $settings{'use_snicar_ad'}       = $nl_flags->{'use_snicar_ad'};
   $settings{'use_century_decomp'}  = $nl_flags->{'use_century_decomp'};
   $settings{'use_crop'}            = $nl_flags->{'use_crop'};
+  $settings{'use_lnd_rof_two_way'} = $nl_flags->{'use_lnd_rof_two_way'};
```

Comment:
Not sure whether it is needed.

Reply:
I followed other feature flag to add the two-way coupling control flag here. I am not sure if this is necessary, @bishtgautam, do you know if we can delete it?